homeassistant/components/plex/__init__.py

```diff
@@ -145,9 +145,13 @@ def _setup_plex(hass, config):
 
     entry.add_update_listener(async_options_updated)
 
-    hass.data[PLEX_DOMAIN][REFRESH_LISTENERS][server_id] = async_track_time_interval(
-        hass, lambda now: plex_server.update_platforms(), timedelta(seconds=10)
+    future = asyncio.run_coroutine_threadsafe(
```

Comment:
We're already in async context so this shouldn't be needed. What is the intent?

If we need to avoid tracking this task we can use `hass.loop.create_task`.

Reply:
To split off a separate thread which listens for websocket messages and acts on them. Future is saved so it can be shut down cleanly when unloading.